app/pages/resource/resource-info/ResourceInfo.js

```diff
@@ -56,7 +61,7 @@ function ResourceInfo({
         </Row>
       </ResourcePanel>
 
-      <Equipment equipment={resource.equipment} />
+      { resource.equipment.length > 0 && (<Equipment equipment={resource.equipment} />) }
```

Comment:
Is `resource.equipment` guaranteed to be an array even if there is no equipment? I would probably first check that it also is an array (e.g. `Array.isArray(resource.equipment) && resource.equipment.length > 0 ...`